docs/contributing/building/nocore-ebpf.md

```diff
@@ -9,22 +9,19 @@
 !!! Introduction
     As you are probably already aware, **Tracee** consists of:
```

Comment:
Since we are not talking about tracee-ebpf, this sentence does not make much sense

Reply:
@AnaisUrlichs That introduction is for Tracee as a whole; removing `-ebpf` part of the naming and inserting `and detects` in the list kinda solves the one binary proposal.

But for sure, it can always be better. What do you suggest?

Reply:
ah never mind